pkg/client/fetch_summary/fetch_summary.go

```diff
@@ -1,17 +1,32 @@
-package client
+package fetch_summary
```

Comment:
would call this package fetch and change inner types to Summary so it would be fetch.ResourceSummary etc'